landoui/assets_src/js/components/Navbar.js

```diff
@@ -0,0 +1,39 @@
+$.fn.landoNavbar = function(){
+  return this.each(function(){
+    let $navbar = $(this);
+
+    // Initialize the responsive menu.
+    $menu = $navbar.find('#Navbar-menu').first();
```

Comment:
Missing lots of `var` declarations; we're leaking lots of globals here.  You'll benefit from adding `"use strict";` at the top of all JS files.